internal/generator/vector/source/http_receiver.go

```diff
@@ -1,15 +1,16 @@
 package source
 
 import (
+	"strings"
+
 	configv1 "github.com/openshift/api/config/v1"
 	logging "github.com/openshift/cluster-logging-operator/apis/logging/v1"
 	"github.com/openshift/cluster-logging-operator/internal/generator/framework"
 	"github.com/openshift/cluster-logging-operator/internal/generator/vector/helpers"
 	"github.com/openshift/cluster-logging-operator/internal/tls"
-	"strings"
 )
 
-func NewHttpSource(id string, input logging.InputSpec, op framework.Options) framework.Element {
+func NewHttpSource(id, inputName string, input logging.InputSpec, op framework.Options) framework.Element {
```

Comment:
I don't recall since this is a subtle modification of my original code, but why would a "source" have an input name?  Looking at the template it should be the name of the secret associated with the input.  I presume the user defines the secret name.  But we should IMO:

* choose a better variable name
* confirm the mount path.  It may be we do some name mangling for the volume/volumeMounts

Reply:
This source is in reference to an HTTP receiver. The input name is the name used when defining the http receiver input. Something like this CLF config,
```
spec:
  inputs:
    - name: input-http1
      receiver:
        http: 
          format: kubeAPIAudit
          receiverPort: 
            name: httpserver1
            port: 443
            targetPort: 8081
```
would have an inputName of `input-http1`.

The mount paths are for the secret that the HTTP receiver service creates. Since I am now making unique names using `resources.GenerateInputServiceName()`, the paths need to reflect the new generated name.